### PR TITLE
i18n(de): add preact integration guide

### PR DESCRIPTION
**Pull request by DerTimonius**
#### Description (required)

supersedes #8146 

This PR adds the missing german translation for the preact integration guide. I also made small change to the english version as the tutorial on the preact website is no longer called Learn Preact in 10 minutes, just Learn Preact.

Added the missing part from the original docs

### CODE REVIEW

File: src/content/docs/de/guides/integrations-guide/preact.mdx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+});
+```
+
+## Optionen
+
+### Kombination mehrerer JSX-Frameworks
+
+Wenn du verschiedene JSX-Frameworks (React, Preact, Solid) im gleichen Projekt verwendest, muss Astro bestimmen, welche JSX-Framework-spezifischen Transformationen für jede deiner Komponenten verwendet werden sollen. Wenn du nur eine JSX-Framework-Integration zu deinem Projekt hinzugefügt hast, ist keine weitere Konfiguration nötig.
+
+Verwende dazu die `include` (erforderlich) und `exclude` (optional) Konfigurationsoption, um zu spezifizieren, welche Dateien zu welchem Framework gehören. Stelle dazu ein Array aus Dateien und/oder Verzeichnissen in der `include`-Option für jedes Framework, das du nutzt, bereit. Platzhalter können verwendet werden, um mehrere Dateipfade anzugeben.
```
Comment (Maxframe) [inline src/content/docs/de/guides/integrations-guide/preact.mdx:174]:
```suggestion
Verwende dazu die `include` (erforderlich) und `exclude` (optional) Konfigurationsoption, um zu spezifizieren, welche Dateien zu welchem Framework gehören. Stelle dazu ein Array aus Dateien und/oder Verzeichnissen in der `include`-Option für jedes Framework, das du nutzt, bereit. Zudem können auch Wildcards verwendet werden, um mehrere Dateipfade anzugeben.
```
Gleiche Argumentation wie bei der SolidJS-Übersetzung (#8624)

**Review by Maxframe — COMMENTED** [review]
Abgesehen von diesen zwei kleinen Vorschlägen, LGTM 💯

**Comment by yanthomasdev** [comment]
Hey @DerTimonius, thanks for opening this PR again, could you make a separate PR for the change to the English version of this page? What I want is this:
1. Remove the change to the English file from this PR.
2. Make a new PR with only the English change, separate from the translation.

**Comment by DerTimonius** [comment]
@yanthomasdev done! https://github.com/withastro/docs/pull/8644

**Review by yanthomasdev — APPROVED** [review]
LGTM! Amazing work peeps! 🙌

**Pull request merged**